packages/framework/src/Model/Product/Collection/ProductParametersBatchLoader.php

```diff
@@ -51,11 +52,11 @@ public function getProductParametersByName(Product $product, DomainConfig $domai
     {
         $key = $this->getKey($product, $domainConfig);
 
-        if (!array_key_exists($key, $this->loadedProductParametersByName)) {
+        try {
+            return $this->localCacheFacade->getItem(static::PARAMETERS_CACHE_NAMESPACE, $key);
+        } catch (LocalCacheException) {
```

Comment:
I wonder - would not be more correct to use hasKey instead of catching exception?

Reply:
There are multiple occurences of this.

Reply:
i did this as an example of possible uses